.github/workflows/build_test.yml

```
@@ -277,3 +277,32 @@ jobs:
         docker run --rm --interactive -v $(pwd):/numpy the_build /bin/bash -c "
           cd /numpy && python3 runtests.py -n -v -- -k test_simd
         "
+
+  sde_simd_avx512_test:
+    needs: [smoke_test]
```

Can we add a comment for people who don't know what sde is, something like
```suggestion
    # Intel provides a Software Development Emulator (SDE) that allows emulating
    # various instruction sets. Use it to check different SIMD commands.
    needs: [smoke_test]
```

---

done